modules/tensor_mechanics/src/materials/ComputeLinearElasticPFFractureStress.C

```diff
@@ -49,54 +65,64 @@ ComputeLinearElasticPFFractureStress::computeQpStress()
   if (c > 1.0)
     cfactor = 0.0;
 
-  // Compute eigenvectors and eigenvalues of ustress
   RankTwoTensor eigvec;
   std::vector<Real> eigval(LIBMESH_DIM);
-  _mechanical_strain[_qp].symmetricEigenvaluesEigenvectors(eigval, eigvec);
-
+  std::vector<Real> eigabs(LIBMESH_DIM);
```

Comment:
Do not use `std::vector` for this! This is a usecase for `RealVectorValue`!

Reply:
This is the old code I should have deleted since eigabs is not used. I will delete this one.

Reply:
I have made both changes and committed to this branch, do you still need me to make any other changes?